Lib/glyphsLib/builder/axes.py

```diff
@@ -165,12 +165,16 @@ def to_designspace_axes(self):
                 continue
         # See https://github.com/googlefonts/glyphsLib/issues/280
         elif font_uses_axis_locations(self.font):
-            # Build the mapping from the "Axis Location" of the masters and instances
+            # If all masters have an "Axis Location" custom parameter, only the values
+            # from this parameter will be used to build the mapping of the masters and instances
             # TODO: (jany) use Virtual Masters as well?
+            #       (jenskutilek) virtual masters can't have an Axis Location parameter.
             mapping = {}
             for master in self.font.masters:
                 designLoc = axis_def.get_design_loc(master)
-                userLoc = axis_def.get_user_loc(master)
+                userLoc = axis_def.get_user_loc_from_axis_location_cp(master)
+                if userLoc is None:
```

Comment:
isn't this check for None-ness ineffective, given that `font_uses_axis_locations` already ensures that all the masters have Axis Location custom parameter and as such will have a non-None userLoc?

Reply:
if anything we want to `assert userLoc is not None`